server/parsec/components/postgresql/user.py

```diff
@@ -107,17 +106,23 @@ def __init__(self, pool: AsyncpgPool, event_bus: EventBus) -> None:
         self.pool = pool
         self.event_bus = event_bus
 
+    async def _check_common_topic(
+        self, conn: AsyncpgConnection, organization_id: OrganizationID
+    ) -> DateTime:
+        common_timestamp = await conn.fetchval(
+            *_q_check_common_topic(organization_id=organization_id.str)
+        )
+        if common_timestamp is None:
+            common_timestamp = DateTime.from_timestamp_micros(0)
```

Comment:
I don't get why the default timestamp here is 0

Reply:
That's the lower date possible, most likely epoch. That makes it a good default when no certificates are present.

Reply:
Ok ^^

But is a situation when there is no certificate at all expected ?

Reply:
yes, when bootstrapping the organization if I remember correctly.